### PR TITLE
Update the CLI docs to display whether a root rotation is in progress

### PR DESCRIPTION
**Pull request by cyli**
...when viewing system info, and TLS info when displaying node info.

Signed-off-by: Ying Li <ying.li@docker.com>

~~This can be merged when https://github.com/docker/cli/pull/44 is merged, which itself can be merged when https://github.com/moby/moby/pull/32875 is merged (for the 17.06 release)~~ Both merged

cc @thaJeztah

cute

### CODE REVIEW

File: docs/reference/commandline/node_ls.md [hunk 1]
```diff
@@ -145,17 +145,19 @@ Placeholder      | Description
 `.Status`        | Node status
 `.Availability`  | Node availability ("active", "pause", or "drain")
 `.ManagerStatus` | Manager status of the node
+`.TLSStatus`     | TLS status of the node (whether it is signed by/trusts the current swarm root CA or if it needs a cert rotation)
```
Comment (mstanleyjones) [inline docs/reference/commandline/node_ls.md]: This is long compared to the others.

**Review by mdlinville — APPROVED** [review]
One comment, but LGTM

File: docs/reference/commandline/node_ls.md (hunk 1 above)
Comment (cyli) [inline docs/reference/commandline/node_ls.md]: Thanks for having a look @mstanleyjones!  Have tried to shorten this to be no longer than the description for `.Self`.

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
Waiting for https://github.com/docker/cli/pull/44 to be merged first

**Review by mdlinville — APPROVED** [review]
LGTM

**Comment by cyli** [comment]
Since https://github.com/moby/moby/pull/33148 has been merged, should we merge this one too?

**Comment by thaJeztah** [comment]
Yes, let's merge this 👍
